include/cell/traits/bmm.hpp

```diff
@@ -10,7 +10,7 @@ using namespace cute;
 namespace tl = tile_layout;
 
 template <typename Element_, typename CtaTileShape,
-          typename WarpArrangement = tiledcuda::cell::TileShape<1, 2>,
+          typename WarpArrangement = tiledcuda::cell::TileShape<2, 2>,
```

Comment:
I’m quite curious about how the default configuration`typename WarpArrangement = tiledcuda::cell::TileShape<2, 2>` was considered?

Reply:
It is not necessary to use small threads setting.